octavia_f5/controller/worker/l2_sync_manager.py

```diff
@@ -209,13 +212,14 @@ def ensure_l2_flow(self, selfips: [network_models.Port], network_id: str, device
                 'store': {'bigip': bigip, 'network': network, 'subnet_id': subnet_ids.pop()},
                 'selfips': selfips_for_host,
             })
-        fs[self.executor.submit(
-            self._do_ensure_l2_flow,
-            data=ensure_l2_flow_data)] = self._bigips
+        if self._bigips:
+            fs[self.executor.submit(
+                self._do_ensure_l2_flow,
+                data=ensure_l2_flow_data)] = self._bigips
 
-        # run VCMP l2 flow for all VCMPs in parallel
+        # run VCMP l2 flow for all VCMP hosts in parallel
```

Comment:
Is it really related to this PR?

Reply:
Yes, this PR is about vCMP hosts. This comment is about vCMP hosts as well. I just thought it could be clearer, so I made this little change.